src/oop_ext/interface/_interface.py

```diff
@@ -354,6 +384,50 @@ def _IsImplementationFullChecking(class_or_instance, interface):
         return True
 
 
+def _IsInterfaceDeclared(class_, interface):
+    """
+    :type interface: Interface or iterable(Interface)
+    :param interface:
+        The target interface(s). If multitple interfaces are passed the method will return True
+        if the given class or instance implements any of the given interfaces.
+    :rtype: True if the object declares the interface passed and False otherwise. Note that
+    to declare an interface, the class MUST have declared
+        >>> ImplementsInterface(Class)
+    """
+    if class_ is None:
+        return False
+
+    is_collection = False
+    if isinstance(interface, (set, list, tuple)):
```

Comment:
```suggestion
    if isinstance(interface, Iterable):
```

More modern :)

Reply:
I noticed that `interface` is never passed as an iterable for this method, so I removed the support for that.